live-examples/html-examples/forms/css/legend.css

```diff
@@ -0,0 +1,19 @@
+.output {
+    font: 1rem 'Fira Sans', sans-serif;
+}
+
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+    url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+input {
+    margin: .4rem;
+}
+
+legend {
+  background-color: #000000;
+  color: #ffffff;
```

Comment:
For hex codes, use the short form: https://mdn.github.io/mdn-fiori/patterns/css/formatting/#hex-color-codes

Reply:
Also, do you think it would be good to have this rule at the top of the file? Since it applies to the element we are demonstrating?